### PR TITLE
Feature/cvsb 8677

### PR DESCRIPTION
**Pull request by castancu**

### CODE REVIEW

File: @Types/ITechRecord.d.ts [hunk 1]
```diff
@@ -82,54 +85,60 @@ interface Axle {
     };
 }
 
-export interface AdrDetails {
```
Comment (vladddragomir) [inline @Types/ITechRecord.d.ts]: why did you remove all the interfaces on adrDetails?

Comment (castancu) [reply @Types/ITechRecord.d.ts]: there is an interface for AdrDetails following the latest api specs, look below you'll find it

File: @Types/ITechRecord.d.ts [hunk 2]
```diff
@@ -82,54 +85,60 @@ interface Axle {
...
+    applicantDetails: {
+        name: string,
+        street: string,
+        town: string,
+        city: string,
+        postcode: string
+    },
+    memosApply?: string[],
+    additionalNotes?:{
+        number?: string[],
```
Comment (vladddragomir) [inline @Types/ITechRecord.d.ts]: this change will affect the mobile app. initially, it was "number": number (it can also be string). the issue is that now this property is an array. why does a property called "number" (not "numbers") is an array and not a single value?

File: @Types/ITechRecord.d.ts [hunk 3]
```diff
@@ -82,54 +85,60 @@ interface Axle {
...
+        guidanceNotes?: string[]
+    },
+    adrTypeApprovalNo?: string,
+    tank?:{
+        tankDetails?:{
```
Comment (vladddragomir) [inline @Types/ITechRecord.d.ts]: this tankDetails object being restructured will also break the mobile app

File: @Types/ITechRecord.d.ts (hunk 2 above)
Comment (castancu) [reply @Types/ITechRecord.d.ts]: This is how the latest excel with the AdrFields looks like. I just followed that, wrote the API specs and implemented them. I think these issues should be discussed with someone else, I'm not in position to answer to this. If you want to take a look at the excel I'm talking about, you'll find it attached in ticket 8714 along with the latest version of the api spec.

Comment (vladddragomir) [reply @Types/ITechRecord.d.ts]: yes, i totally agree. i am just leaving this comment here because @arupdvsa said mobile app won't be affected by the changes. it actually is.

File: @Types/ITechRecord.d.ts (hunk 3 above)
Comment (arupdvsa) [reply @Types/ITechRecord.d.ts]: as discussed this is a minimal change , although understand this will impact VTA mobile app. @vladddragomir  will update the stakeholders.

File: @Types/ITechRecord.d.ts (hunk 2 above)
Comment (arupdvsa) [reply @Types/ITechRecord.d.ts]: I suggested impact will be minimal and insignificant
